master/pkg/searcher/asha.go

```diff
@@ -239,12 +239,36 @@ func (s *asyncHalvingSearch) progress(float64) float64 {
 	if allTrials == s.maxTrials {
 		progress = math.Max(float64(s.trialsCompleted)/float64(s.maxTrials), progress)
 	}
+	// Cap progress at 95% for InvalidHP cases
+	if progress > 1 {
+		progress = float64(0.95)
```

Comment:
blocking: since we're correcting the progress, to we need this?

Reply:
So in the case of ASHA specifically, correcting progress would be difficult as we can’t know whether a trial is closed forever until we finish all possible promotions in a rung (which means trials can start and stop somewhat arbitrarily), estimating progress by units completed is not as feasible. As such, I'm hard capping progress here: see https://determined-ai.slack.com/archives/C019YGJQ0P3/p1607542853041000 for relevant slack discussion.

Reply:
yeah, forgot asha progress is totally different for a minute

Reply:
well that being said, can you drop a descriptive comment explaining why?

Reply:
Yes good idea